flow/project.py

```diff
@@ -933,19 +1091,34 @@ def _create_run_job_operations(self, entrypoint, default_directives, job,
         :rtype:
             Iterator[JobOperation]
         """
+
         for name, op in self.operations.items():
-            if op.eligible(job, ignore_conditions):
-                directives = self._resolve_directives(name, default_directives, job)
-                uneval_cmd = functools.partial(self._run_cmd, entrypoint=entrypoint,
-                                               operation_name=name, operation=op,
-                                               directives=directives, job=job)
-                # Uses a different id than the groups direct id. Do not use this for submitting
-                # jobs as current implementation prevents checking for resubmission in this case.
-                # The different ids allow for checking whether JobOperations created to run directly
-                # are different.
-                job_op = JobOperation(self._generate_id(job, name, index=index), name, job,
-                                      cmd=uneval_cmd, directives=deepcopy(directives))
-                yield job_op
+            filter = self.flow_select[name]
```

Comment:
This doesn't always exist right, looking at the `__init__` function at least. If we allow users to create a `FlowGroup` without explicitly using aggregate features, we shouldn't fail here if they didn't specify those features.

Reply:
Users can create `FlowGroup` instances by using the `make_group` method of the `FlowProject` class (right?).
I have inserted statements when we register group to include these attributes.